dev/puppet-server.conf.sample

```diff
@@ -30,8 +30,8 @@ webserver: {
 # configure the mount points for the web apps
 web-router-service: {
     "puppetlabs.services.ca.certificate-authority-service/certificate-authority-service": "/puppet-ca"
-    "puppetlabs.services.master.master-service/master-service": "/puppet"
-
+    "puppetlabs.services.master.master-service/master-service": {master-routes: "/puppet"
+                                                                 404:           "/" }
```

Comment:
On second thought, are all of these "invalid requests" or "404's" falling under the category of "old puppet agents making previous valid, but now invalid requests?"  If so, I'd make the route identifier named something to indicate that because it's a little confusing to have a route that seems like it's for anything that can't be found (404) but has behavior specific to helping the user upgrade their agent version.